src/utils/Proof.js

```diff
@@ -139,9 +143,11 @@ class Proof {
 
   /**
    * Provides storage path.
+   *
    * @param {Web3} web3 web3 instance of chain from which proof is generated.
    * @param {string} storageIndex Position of storage in the contract.
    * @param {String[]}mappings  list of keys in case storage is mapping.
+   *
```

Comment:
You can create a PR to the developer guidelines to define how the JSDoc should be structured with white lines 😄